docs/application/native/guides/ui/tizen-ws-shell/tzsh-softkey.md

```diff
@@ -0,0 +1,213 @@
+
+- **Opacity State**: You can control the opacity state of the Softkey service window.
+If you set the opacity state to opaque, the background of the Softkey service window changes to 24-bit color opaque window as follows:
+![Expand Off Opaque](./media/tzsh_softkey_expand_off_opaque.png)
+If you set the opacity state to transparent, the background changes to 32-bit colored transparent window as follows:
+![Expand Off Transparent](./media/tzsh_softkey_expand_off_transparent.png)
+
+> **Note**
+>
+> Each state can present differently depending on implementation of the Softkey service.
+> Therefore, preceding example is not represent all of the Softkey service window.
+>
+
+## Prerequisites
+
+To use the functions and structures of the TZSH-Softkey API (in [mobile](../../../api/mobile/latest/group__TIZEN__WS__SHELL__SOFTKEY__MODULE.html) and [wearable](../../../api/wearable/latest/group__TIZEN__WS__SHELL__SOFTKEY__MODULE.html) applications), include the `<tzsh_softkey.h>` header file in your application:
+
+```
+#include <tzsh_softkey.h>
+```
+
+## Create TZSH-Softkey Handler
+
+After you have created the main window of your application, call `tzsh_create()` and `tzsh_softkey_create()` with a native window ID to create the `tzsh_softkey_h` structure (in [mobile](../../../api/mobile/latest/group__TIZEN__WS__SHELL__SOFTKEY__MODULE.html#ga2ee6e5dec1081136fe1139d6a611e58d) and [wearable](../../../api/wearable/latest/group__TIZEN__WS__SHELL__SOFTKEY__MODULE.html#ga2ee6e5dec1081136fe1139d6a611e58d) applications):
+	
+```
+#include <Elementary.h>
+
+static void
+init(const char *name)
+{
+    Evas_Object *main_win;
+
+    /* create main window for your application */
+    main_win = elm-win_util_standard_add(name, name);
+
+    /* set up main window */
+    ...
+    evas_object_show(main_win);
+
+
+    tzsh_h tzsh;
+    tzsh_softkey_h tzsh_sk;
+    tzsh_window tz_win;
+
+    /* Get native window ID of main window */
+    tz_win = elm_win_window_id_get(main_win);
+
+    /* Create tzsh_h structure */
+    tzsh = tzsh_create(tz_win);
+
+    /* Create tzsh_softkey_h structure */
+    tzsh_sk = tzsh_softkey_create(tzsh, tz_win);
+
+    ...
+    /* Do Something Creative */
+```
+
+## Show or Hide Softkey Window
+
+To show or hide the Softkey service window when your application's window is activated, use the following code:
+
+```
+    int ret;
+
+    if (show) {
+        ret = tzsh_softkey_global_show(tzsh_sk); /* To show softkey service window */
+        if (ret != TZSH_ERROR_NONE) {
+            /* error */
+        }
+    } else {
+        ret = tzsh_softkey_global_hide(tzsh_sk); /* To hide softkey service window */
+        if (ret != TZSH_ERROR_NONE) {
+            /* error */
+        }
+    }
+```
+
+## Get Visibility Status of Softkey Window
+
+To get the current visible state of the Softkey service window, call `tzsh_softkey_global_visible_state_get()`. The Softkey service window will be visible or invisible depending on the visible state:
```

Comment:
Please change "visible state" to "visibility state".

Reply:
Done.